rust/agents/relayer/Cargo.toml

```diff
@@ -29,6 +29,8 @@ tracing.workspace = true
 hyperlane-core = { path = "../../hyperlane-core" }
 hyperlane-base = { path = "../../hyperlane-base" }
 hyperlane-ethereum = { path = "../../chains/hyperlane-ethereum" }
+num-derive = "0.3"
```

Comment:
Since we also use `num-traits` and `num-derive` in `hyperlane-core`, we should probably make this a workspace dependency and update `hyperlane-core` to use it as well.